packages/client/src/styles/globals.css

```
@@ -2701,6 +2705,545 @@ body {
+.mari-home-widget {
+  --mari-home-widget-hover-transform: translate3d(0, -0.18rem, 0) scale(1.015);
+  --mari-home-widget-hover-filter: brightness(1.055) saturate(1.045)
+    drop-shadow(
+      0 0.7rem 0.9rem
+        color-mix(
+          in srgb,
+          var(--home-module-accent, var(--marinara-app-accent-solid)) 22%,
+          transparent
+        )
+    );
+  grid-column: span 1;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)